src/Products/Comparison/Service/ComparisonServiceImpl.cs

```diff
@@ -92,37 +97,54 @@ public bool CheckFiles(CompareRequest files)
         }
 
         public CompareResultResponse Compare(CompareRequest compareRequest)
-        {            
+        {
             CompareResultResponse compareResultResponse = CompareTwoDocuments(compareRequest);
             return compareResultResponse;
         }
 
-        public LoadDocumentEntity LoadDocumentPages(string path, string password)
+        public LoadDocumentEntity LoadDocumentPages(string documentGuid, string password, bool loadAllPages)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'LoadDocumentPages' a static method.](https://app.codacy.com/gh/groupdocs-comparison/GroupDocs.Comparison-for-.NET-MVC/pullRequest?prid=5354506)